phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -635,6 +672,9 @@ public boolean verifySingleIndexRow(Result indexRow, IndexToolVerificationResult
         Collections.sort(expectedMutationList, MUTATION_TS_DESC_COMPARATOR);
         Collections.sort(actualMutationList, MUTATION_TS_DESC_COMPARATOR);
         if (verifyType == IndexTool.IndexVerifyType.ONLY) {
+            if (actualMutationList.get(0) instanceof Put) {
```

Comment:
this means only the latest row right?
in that case, a comment would be helpful here 
"Read repair would happen on the latest row hence considering the latest mutation to update the Unverified counters" or something on the similar lines?

Reply:
Yes, look at the previous line : 
Collections.sort(actualMutationList, MUTATION_TS_DESC_COMPARATOR);

Such a comment would be misleading as read repair can happen on any version. If you query with a SCN connection, then read repair will happen on older version. We do read repair only on the latest version as older versions will be always be unverified before newer versions are inserted.

Reply:
I added a comment to emphasize that we verify only the latest version as you suggested